framework/doc/content/getting_started/installation/manual_installation_gcc.md

```diff
@@ -4,16 +4,9 @@
 
 ## Prerequisites
 
-- Cmake 3.4 or greater will be needed for building PETSc. Unless your system is very old, one should
-  be able to use their system's package manager (apt-get, yum, zypper, etc) to install a compatible
-  version of Cmake. For older systems, you will need to obtain cmake source from http://www.cmake.org,
-  and build it appropriately for your system.
-
-- A sane environment. This means having a clean, nothing but the bare minimum as far as available
-  libraries go in your running environment. No additional LD_LIBRARY_PATHs, or other extra PATHs
-  set. No strange UMASK settings. No odd aliases. It might even be best, to create a separate
-  account, strictly for the use of these instructions. I have created an account called 'moose', and
-  will assume you have done the same.
+- Cmake 3.4 or greater will be needed for building PETSc. Unless your system is very old, one should be able to use their system's package manager (apt-get, yum, zypper, etc) to install a compatible version of Cmake. For older systems, you will need to obtain cmake source from http://www.cmake.org, and build it appropriately for your system.
```

Comment:
This may seem pedantic, but Barry Smith didn't like the implication of it seeming like PETSc requires cmake when it most certainly does not. Let's reword this sentence (and the other similar one in this PR) to say "Cmake 3.4 or greater is needed for building some of the optional packages distributed with PETSc that MOOSE requires". That ought to soften the implication just a bit ;)